Repositories/MyPlaceRepository.cs

```diff
@@ -0,0 +1,50 @@
+using rexfinder_api.Migrations;
+using rexfinder_api.Models;
+using rexfinder_api.Repositories;
+
+public class MyPlaceRepository : IMyPlaceRepository
+{
+    private readonly MyPlacesDbContext _context;
+
+    public MyPlaceRepository(MyPlacesDbContext context)
+    {
+        _context = context;
+    }
+    public MyPlace CreateMyPlace(MyPlace newMyPlace)
+    {
```

Comment:
Add this to set the CreatedOn date for the MyPlace:
`newMyPlace.CreatedOn = DateTime.Now.ToString("MM/dd/yyyy, hh:mm:ss tt");`